### PR TITLE
[shopsys] release of version 9.1.1

### PR DESCRIPTION
**Pull request by TomasLudvik**
| Q             | A
| ------------- | ---
|Description, reason for the PR| We have plenty new changes and we want to introduce them to our community.
|New feature| YNo 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,9 +27,99 @@ Packages are formatted by release version. You can see all the changes done to p
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html) as explained in the [Backward Compatibility Promise](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/).
 
-The changelog is generated during the release process using [ChangelogLinker](https://github.com/symplify/changeloglinker) since `v7.0.0-alpha6` release.
-
-<!-- changelog-linker -->
+The changelog is generated during the release process using [Github Changelog Generator](https://github.com/github-changelog-generator/github-changelog-generator) since `v9.1.1` release.
+
+## [v9.1.1](https://github.com/shopsys/shopsys/tree/v9.1.1) (2021-04-14)
```
Comment (grossmannmartin) [inline CHANGELOG.md:32]: Just thinking about the link here. In the previous releases, it was "full changelog" link. 
Now it's a source code in a specific tag. I thought about linking the release on Github, but there is just a link to changelog. 

Do you find it beneficial to have a link to the source code and the next link is the full changelog?

**Review by grossmannmartin — CHANGES_REQUESTED** [review]

File: packages/framework/src/Resources/translations/messages.cs.po [hunk 2]
```diff
@@ -85,9 +85,6 @@ msgstr "Přidat stát"
 msgid "Add item"
 msgstr "Přidat položku"
 
-msgid "Add only to order confirmation page"
```
[the remark this answers is not in the document]
Comment (TomasLudvik) [reply packages/framework/src/Resources/translations/messages.cs.po:88]: fixed

File: upgrade/upgrading-monorepo.md [hunk 3]
```diff
@@ -200,7 +202,8 @@ Typical upgrade sequence should be:
     - `build.dockerfile` should be `docker/Dockerfile`
     - execute `docker-compose up -d --build`, microservices should be up and running
 
-[From v9.1.0 to v9.2.0-dev]: https://github.com/shopsys/shopsys/compare/v9.1.0...master
+[From v9.1.1 to v9.1.2-dev]: https://github.com/shopsys/shopsys/compare/v9.1.1...tl-release-9-1-1
```
[the remark this answers is not in the document]
Comment (TomasLudvik) [reply upgrade/upgrading-monorepo.md:205]: fixed

File: upgrade/UPGRADE-v9.1.2-dev.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# [Upgrade from v9.1.1 to v9.1.2-dev](https://github.com/shopsys/shopsys/compare/v9.1.1...tl-release-9-1-1)
```
[the remark this answers is not in the document]
Comment (TomasLudvik) [reply upgrade/UPGRADE-v9.1.2-dev.md:1]: fixed

File: upgrade/UPGRADE-v9.1.1.md [hunk 5]
```diff
@@ -1,24 +1,24 @@
-# [Upgrade from v9.1.0 to v9.1.1-dev](https://github.com/shopsys/shopsys/compare/v9.1.0...9.1)
+# [Upgrade from v9.1.0 to v9.1.1](https://github.com/shopsys/shopsys/compare/v9.1.0...9.1)
```
[the remark this answers is not in the document]
Comment (TomasLudvik) [reply upgrade/UPGRADE-v9.1.1.md:1]: fixed

File: upgrade/UPGRADE-v9.1.1.md [hunk 6]
```diff
@@ -1,24 +1,24 @@
...
 
-This guide contains instructions to upgrade from version v9.1.0 to v9.1.1-dev.
+This guide contains instructions to upgrade from version v9.1.0 to v9.1.1.
 
 **Before you start, don't forget to take a look at [general instructions](https://github.com/shopsys/shopsys/blob/master/UPGRADE.md) about upgrading.**
 There you can find links to upgrade notes for other versions too.
 
 - sanitize non-printable search text ([#2174](https://github.com/shopsys/shopsys/pull/2174))
-    - see #project-base-diff to update your project
+    - see [project-base-diff](https://github.com/shopsys/project-base/commit/2bdd245c6598c12ef1e18981f42effa04cc26c92) to update your project
```
[the remark this answers is not in the document]
Comment (TomasLudvik) [reply upgrade/UPGRADE-v9.1.1.md:10]: fixed

File: CHANGELOG.md (hunk 1 above)
Comment (TomasLudvik) [reply CHANGELOG.md]: I have updated it to previous style

**Review by grossmannmartin — APPROVED** [review]

**Review by sslt — APPROVED** [review]

**Pull request merged**